tests/bootstrap.php

```diff
@@ -6,9 +6,10 @@
 
 // autoload rector first, but with local paths
 // build preload file to autoload local php-parser instead of phpstan one, e.g. in case of early upgrade
-exec(PHP_BINARY.' vendor/rector/rector-src/build/build-preload.php '.__DIR__.'/../');
+$preloadPath = __DIR__ . '/../';
+exec(PHP_BINARY . ' vendor/rector/rector-src/build/build-preload.php ' . $preloadPath);
 sleep(3);
 
-require __DIR__ . '/../preload.php';
+require $preloadPath . 'preload.php';
```

Comment:
This hides the absolute path and breaks PHPStorm autocomplete. 
In this case, just add it to `phpstan.neon`, as this path is generated dynamically to test new php-parser 5.